### ISSUE TITLE
Call symfony in vagrant

### ISSUE THREAD

**Issue by st-schneider** [TheNouillet/symfony-vscode#33]
would be nice if this would be an option.

**Comment by st-schneider** [comment]
ok I found the docker example and adjusted my command but i get the following
```                             
  The "-t" option does not exist.  
                                   
debug:router [--show-controllers] [--format FORMAT] [--raw] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] <command> [<name>]


```

**Comment by TheNouillet** [comment]
Hello,

What command did you put in the `Shell Command` setting ? When calling the `debug:router` command, the extension doesn't add any argument beside `--format=json`. Do you have the same problem when only refreshing services ?

**Comment by st-schneider** [comment]
```
    "symfony-vscode.consolePath": "/var/www/app/console",
    "symfony-vscode.shellExecutable": "/bin/bash",
    "symfony-vscode.shellCommand": "vagrant ssh -t -c 'cd /var/www && php \"$@\"' d8d52c5 -- "
```
Yes I get that for all the view, then with debug:container

**Comment by st-schneider** [comment]
btw doesnt matter if the -t for vagrant is there or not

**Comment by igorsinkovec** [comment]
I have the same problem; both on load and when refreshing services. I did not include `-t` switch in `symfony-vscode.shellCommand` config.

**Comment by igorsinkovec** [comment]
Solved by ssh-ing to the Vagrant box directly. First I created a ssh config file from Vagrant config:
`vagrant ssh-config > vagrant-ssh-config`
The shell command config then looks like this:
`"symfony-vscode.shellCommand": "cd /path/to/vagrant-dir; ssh -F vagrant-ssh-config <vagrant-box-name>; 'cd /path/to/symfony-app; php \"$@\"' -- "`

**Comment by st-schneider** [comment]
[2019-04-10 17:23:18.826] [exthost] [error] [TheNouillet.symfony-vscode] provider FAILED
[2019-04-10 17:23:18.826] [exthost] [error] TypeError: Cannot read property 'find' of undefined

**Comment by st-schneider** [comment]
this is what I get with that

**Comment by igorsinkovec** [comment]
Yes, I get that too, but I don't think it's related.

**Comment by st-schneider** [comment]
shouldn't it be?:
ssh -F vagrant-ssh-config <vagrant-box-name> 'cd /path/to/symfony-app; php \"$@\"'

**Comment by igorsinkovec** [comment]
> shouldn't it be?:
> ssh -F vagrant-ssh-config 'cd /path/to/symfony-app; php "$@"'

Yes, you're correct, that's a typo. Will edit my previous comment. Thanks for noticing.

**Comment by igorsinkovec** [comment]
> shouldn't it be?:
> ssh -F vagrant-ssh-config 'cd /path/to/symfony-app; php "$@"'

Not in my environment. SSH does not connect without the host machine name.

**Comment by st-schneider** [comment]
ehh of coursem my point was actuall the ;

**Comment by st-schneider** [comment]
ssh -t -F vagrant-ssh-config default 'cd /var/www; php \`$@\`' --
but then I get 
[2019-04-11 09:38:29.810] [exthost] [error] [TheNouillet.symfony-vscode] provider FAILED
[2019-04-11 09:38:29.811] [exthost] [error] Error: Illegal argument: character must be non-negative

**Comment by st-schneider** [comment]
This works for me.
```
"symfony-vscode.shellCommand": "cd /path/to/vagrant-dir && vagrant ssh-config > vagrant-ssh-config && ssh -t -F vagrant-ssh-config <vagrant-box-name> 'cd /path/to/symfony-app && php `$@`' ",
```
